detectron2/config/defaults.py

```diff
@@ -177,7 +177,9 @@
 # list[float], the angle in degrees, for each input feature map.
 # ANGLES[i] specifies the list of angles for IN_FEATURES[i].
 _C.MODEL.ANCHOR_GENERATOR.ANGLES = [[-90, 0, 90]]
-
+# Offset of anchor relave to tl corner of the pixel
+# Allowed values are floats in [0, 1] range inclusive
```

Comment:
- Please expand "tl" to "top-left".
- Specify the units for this option: "fraction of feature map stride (e.g., 0.5 means half stride)"

Reply:
Fixed